src/Sentry/Platforms/Cocoa/BindableSentryOptions.cs

```diff
@@ -26,6 +26,7 @@ public class NativeOptions
         public bool? EnableUIViewControllerTracing { get; set; }
         public bool? EnableUserInteractionTracing { get; set; }
         public bool? EnableTracing { get; set; }
+        public bool? SuppressSegfaults { get; set; }
```

Comment:
I think the SuppressSegfaults stuff should go in separate PR:
- It needs it's own entry in the changelog (letting people know about the option, and the fact that the default is changing vs 5.1.1)
- Easier to review both PRs 
- Easier to work out what had to change to enable a feature/fix , if someone comes back to this PR in 12-18 months
- Easier to revert, if we need to

It could be done as a fork of this branch, and we review/merge after this branch is reviewed/merged... but definitely tidier if it's in it's own PR.

Reply:
Moved to https://github.com/getsentry/sentry-dotnet/pull/3998:
- https://github.com/getsentry/sentry-dotnet/pull/3998